pyuvdata/uvdata/uvdata.py

```diff
@@ -770,6 +771,17 @@ def __init__(self):
 
         super(UVData, self).__init__()
 
+    @staticmethod
```

Comment:
Yeah -- it doesn't use the class. Usually, you'd have a constructor that's a fairly simple function, where you modify some of the input params a bit and send them through to the `cls` to be constructed (and validated etc.). In this case, I can't just do a `cls(param1, param2, ....)` at the end, because that's not how pyuvdata was written. Since the construction is so complex, I wanted to put it in another module to keep it tidy. But then the function in that module just imports UVData directly to construct the object, so it doesn't need to be passed the `cls`. Since we don't need `cls`, it's best to make it a `staticmethod`